srcpkgs/xf86-video-amdgpu/template

```diff
@@ -1,21 +1,20 @@
 # Template file for 'xf86-video-amdgpu'
 pkgname=xf86-video-amdgpu
-version=19.1.0
-revision=2
+version=21.0.0
+revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config"
-makedepends="xorgproto eudev-libudev-devel libpciaccess-devel
- libdrm-devel pixman-devel MesaLib-devel xorg-server-devel"
+makedepends="xorgproto eudev-libudev-devel libpciaccess-devel libdrm-devel
+ pixman-devel MesaLib-devel xorg-server-devel xorg-util-macros"
 depends="virtual?xserver-abi-video-24_1 mesa-dri"
 short_desc="Xorg AMD Radeon RXXX video driver (amdgpu kernel module)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://wiki.freedesktop.org/xorg"
 distfiles="${XORG_SITE}/driver/${pkgname}-${version}.tar.bz2"
-checksum=4f0ea4e0ae61995ac2b7c72433d31deab63b60c78763020aaa1b28696124fe5d
+checksum=607823034defba6152050e5eb1c4df94b38819ef764291abadd81b620bc2ad88
 
 lib32disabled=yes
-CFLAGS="-fcommon"
 LDFLAGS="-Wl,-z,lazy"
 
 case "$XBPS_TARGET_MACHINE" in
```

Comment:
Given https://gitlab.freedesktop.org/xorg/driver/xf86-video-amdgpu/-/commit/6ed48634443e15a45f48e3a4ddf91e46041ad38f , please try to remove this broken (even if the driver is kinda useless for that target).